kafka-plugins-0.8/pom.xml

```diff
@@ -51,6 +52,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
just use the non-SNAPSHOT version?
Is the SNAPSHOT version even still available?

Reply:
it seems to compile fine. I'll update though

Reply:
or maybe its my local

Reply:
ya, I suspect you have the dependent jar cached locally.

Reply:
fixed